mitsurunner.yaml

```
@@ -97,6 +109,28 @@ sensor:
     state_topic: $topic_outdoor
     filters:
       - filter_out: nan
+      - sliding_window_moving_average:
+          window_size: 360  #30min
+          send_every: 6     #30s
+          send_first_at: 2  #10s
+    on_raw_value:
+      then:
+        lambda: !lambda |-
+            // Calculate exponential moving average for outdoor temperature
+            // Alpha is adjustable, similar to one in the built-in exponential_moving_average filter
+            static const float alpha = 0.1;
+            static bool first_time = 1;
+            // Do not use filter until first real (not NaN) value received
+            if(first_time) {
+                if (!isnan(x)) {
+                  id(G_outdoor_temp_filtered) = x;
+                  first_time = 0;
```

And false here for first_time 
```suggestion
                  first_time = false;
```